models/EventModel.ts

```diff
@@ -67,11 +67,11 @@ export class EventModel extends BaseEntity {
   @OneToMany((type) => ExpressCheckinModel, (expressCheckin) => expressCheckin.event, { cascade: true })
   expressCheckins: ExpressCheckinModel[];
 
-  @Column('uuid', { nullable: true })
-  discordEvent: Uuid;
+  @Column('varchar', { nullable: true })
+  discordEvent: string;
```

Comment:
In types/index.ts, we define Uuid with `type Uuid = string` and we just use it for clarification on the context we use it. As a result, I think it is better to leave Uuid as type here as well

Reply:
So to clarify, the column type can be `varchar`, but it should be `discordEvent: Uuid`?